connector.yaml

```diff
@@ -0,0 +1,142 @@
+version: "1.0"
+specification:
+  name: file
+  summary: A file source and destination plugin for Conduit.
+  description: "## How it works\nThe Source connector listens for changes appended to the source file and \nsends records with the changes.\nThe Destination connector receives records and writes them to a file.\n\n### Source\nThe Source connector only cares to have a valid path, even if the file \ndoesn't exist, it will still run and wait until a file with the configured\nname is there, then it will start listening to changes and sending records.\n\n### Destination\nThe Destination connector will create the file if it doesn't exist, and \nrecords with changes will be appended to the destination file when received."
```

Comment:
Oh and we should reformat this into a multi-line string instead of using these pesky `\n`.